spacecmd/src/bin/spacecmd

```diff
@@ -28,33 +29,44 @@ import logging
 import os
 import re
 import sys
+
 try:
     from xmlrpc import client as xmlrpclib
 except ImportError:
     import xmlrpclib
+
 import codecs
 import locale
 import argparse
-try: # python 3
-    from configparser import SafeConfigParser
-except ImportError: # python 2
-    from ConfigParser import SafeConfigParser
+
+try:
+    try:  # python >= 3.12
+        from configparser import ConfigParser
+    except ImportError:  # python 3
+        # pylint: disable-next=deprecated-class
+        from configparser import SafeConfigParser as ConfigParser
+except ImportError:  # python 2
+    from ConfigParser import SafeConfigParser as ConfigParser
```

Comment:
`SafeConfigParser` is an deprecated alias of `ConfigParser` since Python 3.2. We can just use the "new" (14 years old) name for all Python 3 versions.

```suggestion
try:
    from configparser import ConfigParser
except ImportError:  # python 2
    from ConfigParser import SafeConfigParser as ConfigParser
```